### PR TITLE
[TT-9414] Update documentation from gateway:master

### PR DESCRIPTION
**Pull request by Tyk-ITS**
Triggered by: titpetric

JIRA: https://tyktech.atlassian.net/browse/TT-9414

### CODE REVIEW

File: tyk-docs/assets/others/dashboard-admin-swagger.yml [hunk 1]
```diff
@@ -387,14 +387,14 @@ paths:
                 - access_rights:
                     5fa2db834e07444f760b7ceb314209fb:
                       allowed_urls: []
-                      api_id: 5fa2db834e07444f760b7ceb314209fb
-                      api_name: API 2
+                      apiid: 5fa2db834e07444f760b7ceb314209fb
```
Comment (dcs3spp) [inline tyk-docs/assets/others/dashboard-admin-swagger.yml:390]: Should this use snake casing or camel casing?

**Comment by dcs3spp** [comment]
@titpetric Many thanks for testing and submitting. Is this intended for release 5.2?

**Review by dcs3spp — COMMENTED** [review]
Is PR testing?

What is the reason for renaming api_id to apiid and api_name to apiname?
Does th PR target release 5.2?

File: tyk-docs/assets/others/dashboard-admin-swagger.yml (hunk 1 above)
Comment (titpetric) [reply tyk-docs/assets/others/dashboard-admin-swagger.yml:390]: out of scope, it reflects master

**Pull request closed without merging**